www/site/content/docs/chef-workstation/about.md

```diff
@@ -6,21 +6,37 @@ title = "About Chef Workstation"
     weight= "30"
 +++
 
-Start your infrastructure automation quickly and easily with Chef Workstation. Chef Workstation give you everything you need to get started with Chef--ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software--all in one easy-to-install package.
+Start your infrastructure automation quickly and easily with Chef Workstation.
+Chef Workstation give you everything you need to get started with Chef--ad hoc
+remote execution, remote scanning, configuration tasks, cookbook creation tools
+as well as robust dependency and testing software--all in one easy-to-install
+package.
 
-Chef Workstation replaces ChefDK, combining all the existing features with the added Chef Workstation feature of running ad-hoc tasks. In the future, Chef will develop and improve Chef Workstation and maintain ChefDK in its current state.
+Chef Workstation replaces ChefDK, combining all the existing features with the
+new features such as ad-hoc task support and the new Chef Workstation desktop
+application.  Chef will continue to maintain ChefDK, but new development will
+take place in Chef Workstation without backporting features.
 
 ## Open Source
 
-We're keeping the tradition of open source development in Chef. You'll find the Chef Workstation source code on [GitHub](https://github.com/chef/chef-workstation). We're releasing Chef Workstation under the open source [Apache 2.0 license](https://github.com/chef/chef-workstation/blob/master/LICENSE).
+We're keeping the tradition of open source development in Chef. You'll find the
+Chef Workstation source code on
+[GitHub](https://github.com/chef/chef-workstation). We're releasing Chef
+Workstation under the open source [Apache 2.0
+license](https://github.com/chef/chef-workstation/blob/master/LICENSE).
 
 ### Contributing to Chef Workstation
 
-We always welcome (and deeply appreciate!) new contributions to the project. The best way to start contributing to Chef Workstation is to provide us with in-depth feedback by creating GitHub issues.
+We always welcome (and deeply appreciate!) new contributions to the project.
+The best way to start contributing to Chef Workstation is to provide us with
+in-depth feedback by creating GitHub issues.
 
-See the guidelines for [community contributions](https://docs.chef.io/community_contributions.html) and our [community guidelines](https://docs.chef.io/community_guidelines.html) for keeping the development of the project awesome for all contributors.
+See the guidelines for [community
```

Comment:
id like the link text to be `community contribution guidelines` ..
so maybe it becomes... `See the [community contribution guidelines](<link>) and our ...`